src/main/kotlin/com/sourcegraph/cody/commands/ui/CommandsContextMenu.kt

```diff
@@ -16,15 +16,17 @@ class CommandsContextMenu {
       for (commandId in CommandId.values()) {
         val actionId = "cody.command.$commandId"
         val existingAction = actionManager.getAction(actionId)
-        if (existingAction != null) {
-          continue
-        }
         val action: DumbAwareAction =
             object : DumbAwareAction(commandId.displayName) {
               override fun actionPerformed(e: AnActionEvent) {
                 executeCommand(commandId)
               }
             }
+        if (existingAction != null) {
```

Comment:
I would like if/else more there as neither logic branch is more special, but I guess it's purely personal preference.
LGTM

Reply:
+1